src/pages/AppDetail.svelte

```diff
@@ -81,4 +82,6 @@
       </li>
     {/each}
   </ul>
+{:else}
```

Comment:
One other small comment in this file -- it looks like the title in this page is outside of the `if` statement, so if it fails there will be two `<h1>` tags. Can you pull it into the conditional so it's consistent?